utils/utils.go

```diff
@@ -0,0 +1,126 @@
+package utils
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"io"
+	"os"
+	"strconv"
+	"net/url"
+	"reflect"
+	"bootcamp/domain/model"
+	"github.com/spf13/viper"
+	"gopkg.in/mgo.v2/bson"
+)
+
+/*
+GetEnvVar gets a given envar by key
+*/
+func GetEnvVar(key string) (string, error) {
```

Comment:
For configuration we would usually read the config file or env variables and insert them into a struct that could be used "anywhere", and config handling usually goes into its own package as well

Reply:
*